util/src/main/java/io/zeebe/util/sched/ActorJob.java

```diff
@@ -44,6 +45,7 @@ void execute(ActorThread runner) {
       }
 
     } catch (Throwable e) {
+      LoggerFactory.getLogger(ActorJob.class).error("FAILED!", e);
```

Comment:
Can we make this a constant? Also do we need to catch `Throwable` here? I know you didn't add this but maybe you know if it's safe to go to `Exception`?